src/test/java/org/stellar/sdk/SorobanServerTest.java

```diff
@@ -555,7 +555,7 @@ public void testGetEvents() throws IOException, SorobanRpcErrorResponse {
             .build();
     GetEventsRequest getEventsRequest =
         GetEventsRequest.builder()
-            .startLedger("100")
+            .startLedger(100L)
```

Comment:
could you add an assertion on `GetEventsResponse.EventInfo.ledger` marshaling correctly as a number now from raw json/rpc payload also?

Reply:
We already have such an assertion. https://github.com/stellar/java-stellar-sdk/blob/3d72a9b78bb053ce016476a841b35acf43d5f37a/src/test/java/org/stellar/sdk/SorobanServerTest.java#L591